docs/framework/winforms/advanced/effects-of-modifying-base-form-appearance.md

```diff
@@ -1,24 +1,22 @@
 ---
 title: "Effects of Modifying a Base Form's Appearance"
 ms.date: "03/30/2017"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "parent forms [Windows Forms]"
   - "inherited forms [Windows Forms], modifications to base form"
   - "Windows Forms, base form appearance"
   - "base forms"
   - "inheritance [Windows Forms], forms"
 ms.assetid: 1c3f2b29-a05c-4c6f-aa1a-4e66b94f343a
 ---
-# Effects of Modifying a Base Form's Appearance
-During application development, you may often need to change the appearance of the base form from which other forms in the project (or in other projects) are inheriting.  
-  
- At design time, changes to the base form's appearance (be it the setting of properties or the addition and subtraction of controls) are reflected on inherited forms when the project containing the base form is built. It is not sufficient for you to simply save the changes to the base form. To build a project, choose **Build** from the **Build** menu.  
-  
-> [!NOTE]
```

Comment:
I'm curious, why are these notes removed?

Reply:
I removed a lot of these notes from the VS docs too. IMO they take up valuable real estate and appear prominently, but are actually not even relevant in most cases. The thing that different development settings affects the most is keyboard shortcuts, and I think that most of these topics don't even use keyboard shortcuts. I'm guessing these notes were originally added as a sort of future-proofing or cover-our-bases thing. If you feel that they are important, perhaps we put a note in the top-level topic of each section instead of each and every topic.